### DISCUSSION TITLE
Meaning of different Q-value columns

### DISCUSSION THREAD

**Discussion by buijt** [vdemichev/DiaNN#780] [Q&A] [answered]
Hello.

I read through the README and previous discussions and I did not understand the explanation of what the different q-value columns in the DIANN_report.tsv output represent.

From the README:

> - **Global** refers to a global q-value, that is calculated for the entire experiment
> - **Lib** refers to the respective value saved in the spectral library, e.g. Lib.Q.Value means q-value for the respective library precursor

I see from https://github.com/vdemichev/DiaNN/discussions/479 that Q-values are CScores from the neural network, i.e. the confidence score of the identification. The README also mentions that the Q-value is similar to a false-discovery rate, so we use the q-value in order to control the amount of true / false positive precursor hits from the data.

However, there are several Q.Value columns in DIA-NN output and I would like to understand what they represent. I have some guesses.

- **Q.Value** must be the raw CScores from the neural network, and users cannot do anything with this value
- **Global.Q.Value** is the confidence of a precursor being a true hit in at least one sample in the data
- **Lib.Q.Value** is the confidence of the precursor existing in the spectral library. For searches in library-free mode (with FASTA sequences), this means that the precursor is confidently predicted from in-silico digestion. Is this correct understanding?
- **Lib.PG.Q.Value** and **PG.Q.Value** I would guess is the confidence of the precursor assignment to a specific protein group. For example, a precursor may be a true hit (based on Lib.Q.Value) but if confidence in the assignment of that precursor to a specific protein is low, then the Protein.Q.Value may be high (insignificant). The "standard" workflow [suggests](https://github.com/vdemichev/DiaNN/discussions/677) filtering on this column because we would not want to falsely report quantitation for an uncertain protein. Is my understanding correct?
- I do not see any reference to the meaning of **Protein.Q.Value**. Could you explain what this represents?

I appreciate any definitions you could provide. Thank you.

**Comment by vdemichev** [comment, marked as answer]
q-values can be calculated based on c-scores (see the mProphet work), which is the case for Q.Value, but they are not c-scores.
Lib.Q.Value is the Global.Q.Value calculated during empirical library generation.
PG q-values reflect the confidence of stating that at least one of the peptides matched to the protein is detected correctly.
Protein.Q.Value - same as PG, but using proteotypic peptides only.

**Comment by buijt** [comment]
Thank you for your prompt response. 

I wanted to also ask, with MBR the DIANN_report.tsv has already applied the 0.01 Q-value filter on precursors using the Lib.Q.Value but does not apply the filter on PG/Protein q values. Why are the PG q-values over 0.01 still included in the report?

**Reply by vdemichev** [reply]
MBR indeed applies Lib.Q.Value <= 0.01 filter automatically. But the main report is never filtered at the protein level, hence all protein q-values (run-specific, lib or global) can be higher than 0.01.

**Comment by wiraki** [comment]
I wanted to add a couple of questions to this. 

1) When MBR is on, why is it better to look at the Lib.(PG.)Q.Value instead of the Global? I struggle to grasp it.

2) When MBR is on, matrices are still created from main report based on Global q-values, not Lib. Is that correct?

**Reply by vdemichev** [reply]
1. As Lib.PG.Q.Value indicates that a protein has been detected reliably during the first pass - which means the protein is present in the experiment. Hence this filtering is sufficient.
2. No, based on Lib.
